linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_sysadmin_actions/ansible/shared.yml

```diff
@@ -0,0 +1,43 @@
+# platform = multi_platform_rhel
+# reboot = false
+# complexity = low
+
+- name: Search /etc/audit/rules.d for other sudoers audit rules
+  find:
+    paths: /etc/audit/rules.d
+    recurse: false
+    contains: -k sudoers$
+    patterns: '*.rules'
+  register: find_sudoers
+  tags:
+    @ANSIBLE_TAGS@
+- name: If existing sudoers ruleset not found, use /etc/audit/rules.d/privileged.rules as the recipient for the rule
+  set_fact:
+    all_files:
+    - /etc/audit/rules.d/privileged.rules
```

Comment:
Bash remediation creates file with `$key.rules` name if it can't find the string in any file. It would be great, if it's consistent, so `/etc/audit/rules.d/actions.rules` is better.